torch/ao/quantization/backend_config/onednn.py

```diff
@@ -0,0 +1,253 @@
+import torch
+import torch.nn as nn
+import torch.nn.intrinsic as nni
+import torch.nn.functional as F
+import torch.nn.quantized._reference as nnqr
+from ._common_operator_config_utils import (
+    _get_binary_op_configs,
+    _get_bn_configs,
+    _get_cat_config,
+    _get_conv_configs,
+    _get_default_op_configs,
+    _get_embedding_op_configs,
+    _get_fixed_qparams_op_configs,
+    _get_linear_configs,
+    _get_rnn_op_configs,
+    _get_share_qparams_op_configs,
+)
+from .backend_config import (
+    BackendPatternConfig,
+    BackendConfig,
+    DTypeConfig,
+    ObservationType,
+)
+from ..fuser_method_mappings import (
+    reverse_sequential_wrapper2,
+    reverse3,
+)
+
+
+# ===================
+# |  DTYPE CONFIGS  |
+# ===================
+
+# Based on fbgemm's configs
```

Comment:
Do we have to copy all the fbgemm configs here? onednn doesn't support all these configs, e.g., it doesn't support fp16 and quint4x2. How about only adding those onednn supports?

Reply:
OK I will fix these later.